docs/source/techniques/index.rst

```diff
@@ -17,3 +17,4 @@ Techniques
    :maxdepth: 1
 
    *
+test_workspacehistorygeneration
```

Comment:
Should this be here?

Reply:
Thank's for spotting this. Funny how these things can slip in.